src/utils.ts

```diff
@@ -416,3 +416,228 @@ export function patchMatrixClient() {
     patchMatrixClientForConciseExceptions();
     patchMatrixClientForRetry();
 }
+
+/**
+ * Common duration units, used for commands that accept durations such as "1 day".
+ */
+export enum DurationUnit {
```

Comment:
I feel uncomfortable with putting in `src/utils.ts` this massive 200 lines long error prone duration handling code given we've had problems with rolling our own time stuff in other projects

Reply:
What would you suggest?

Reply:
We could also use https://www.npmjs.com/package/parse-duration, I suppose.

Reply:
fwiw, https://github.com/wanasit/chrono seems to work fairly well in logseq (speaking as an end user of logseq)

Reply:
Ah, well, I've used parse-duration + humanize-duration.